src/handle-language-definitions.js

```diff
@@ -0,0 +1,9 @@
+import { addChangeLanguageTask } from './actions/task-definitions'
+
+export default function handleLanguageDefinitions(store) {
+  const state = store.getState();
+  const { languageDefinitions } = state;
+  for (const language of Object.values(languageDefinitions)) {
+    addChangeLanguageTask(language.languageId, language.displayName, language.keybinding);
```

Comment:
Launched a notebook, tried typing `p` but didn't get python. Could we also make sure to add the language keybinding?

Reply:
I think this is fixed now.

Reply:
ah, i'm hitting this as well when i try to go js->py or py->js, but *not* if i go js->md->py or vice versa; i think that #1020 broke this. let's take that a separate issue

Reply:
filed at #1041